CHANGELOG-1.15.md

```diff
@@ -167,38 +167,10 @@ filename | sha512 hash
 
 ### Action Required
 
-* ACTION REQUIRED  The deprecated flag --conntrack-max has been removed from kube-proxy. Users of this flag should switch to --conntrack-min and --conntrack-max-per-core instead. ([#78399](https://github.com/kubernetes/kubernetes/pull/78399), [@rikatz](https://github.com/rikatz))
+* ACTION REQUIRED: The deprecated flag --conntrack-max has been removed from kube-proxy. Users of this flag should 
+switch to --conntrack-min and --conntrack-max-per-core instead. ([#78399](https://github.com/kubernetes/kubernetes/pull/78399), [@rikatz](https://github.com/rikatz))
 * ACTION REQUIRED: kubeadm: the mixture of "--config" and "--certificate-key" is no longer allowed. The InitConfiguration and JoinConfiguration objects now support the "certificateKey" field and this field should be used instead of the command line argument in case a configuration file is already passed. ([#78542](https://github.com/kubernetes/kubernetes/pull/78542), [@neolit123](https://github.com/neolit123))
-* Azure cloud provider could now be configured by Kubernetes secrets and a new option `cloudConfigType` is introduced, whose candicate values are `file`, `secret` and `merge` (default is `merge`). ([#78242](https://github.com/kubernetes/kubernetes/pull/78242), [@feiskyer](https://github.com/feiskyer))
-    * action required:
-    * Since Azure cloud provider would read Kubernetes secrets, the following RBAC should be configured:
```

Comment:
some mention of the new permissions required should be kept. this could be switched to two `kubectl create clusterrole` and `kubectl create clusterrolebinding` commands

Reply:
also, the doc should indicate this is only required if the cloudConfigType is `secret`, and the example role should only grant access to the specified secret (so should probably be a role and rolebinding). cc @feiskyer for input on which secret is required.

Reply:
when `cloudConfigType` is configured to secret or merge (this is default value), then the role and rolebing are required.

Note that the secret is a serialized version of azure.json file with key `cloud-config`. And the secret name is `azure-cloud-provider` in `kube-system` namespace.

Reply:
@cwdsuzhou Could you help to add the above comments into release notes.

@liggitt the yaml contents are supported in release 1.12, and the yaml are actually wrap in ```. For 1.15, seems this is not supported anymore. The generated contents are started with `*` for each line.

Reply:
@feiskyer  I have added `Note that the secret is a serialized version of azure.json file with key cloud-config. And the secret name is azure-cloud-provider in kube-system namespace.` to the release notes. Is it ok now?

Reply:
Thanks. LGTM